src/frontend/js/widgets/Dashboard/components/TeacherDashboardCourseSidebar/index.spec.tsx

```diff
@@ -17,7 +17,11 @@ import {
 import { createTestQueryClient } from 'utils/test/createTestQueryClient';
```

Comment:
In the commit [5124f166585fcd67991045dd983c90fb03f7e47b](https://github.com/openfun/richie/commit/5124f166585fcd67991045dd983c90fb03f7e47b) message :

`s/oranization's/organization's`
`s/in coure course sidebar/in course sidebar`